admin/app/controllers/cache/ImageServices.scala

```diff
@@ -2,53 +2,31 @@ package controllers.cache
 
 import java.net.URI
 
-import conf.AdminConfiguration.{fastly, imgix}
-import play.api.libs.json.{JsObject, JsString}
-import play.api.libs.ws.{WSAuthScheme, WSClient}
-import views.support.ImgSrc.tokenFor
-import views.support.ImageUrlSigner.sign
+import conf.AdminConfiguration.fastly
+import conf.Configuration.environment.stage
+import play.api.libs.ws.WSClient
 
-class ImageServices(wsClient: WSClient) {
+object ImageServices {
 
   // none of the stuff here is a state secret.
   // it is all authenticated
 
-  private val iGuimCoUk = "YjEge7vUbdglYXtZ56cU1"
+  private val iGuimCoUk = "1L0HRheo6sMtfQHnY1FU6C"
+  private val iGuimCodeCoUk = "5CSDV7WcKwnIIHipZzt3po"
```

Comment:
I would rename ``iGuimCoUk and `iGuimCodeCoUk` :)

Reply:
Do you mean to fastlyioguimcouk or something similar? That domain should be deprecated soon, now that we have migrated i.guim over to it. fastly-io-code is still in use, but I've moved i.guimcode over to that service so we can stop using that too.